frontend/src/Settings/General/UpdateSettings.js

```diff
@@ -83,8 +85,10 @@ function UpdateSettings(props) {
                 type={inputTypes.CHECK}
                 name="updateAutomatically"
                 helpText={translate('UpdateAutomaticallyHelpText')}
+                helpTextWarning={updateMechanism.value === 'docker' ? translate('AutomaticUpdatesDisabledDocker', { appName: 'Sonarr' }) : undefined}
                 onChange={onInputChange}
                 {...updateAutomatically}
+                isDisabled={!canUpdateAutomatically}
```

Comment:
Actually, as mentioned by @bakerboy448 script updating is technically possible, so this is wholly accurate and would disable that ability.

Reply:
If the update mechanism is `docker`, the user either needs to update via docker (outside of Sonarr) or as baker mentioned before, they could use a script to update docker through compose or some other tooling.

Disabling this means they would be unable to use a script to do the updating automatically.

Reply:
> How can one update with docker compose from inside of a running container?

for example, on identifying an update you could call a script that triggers another container (watchtower) to update this one.